### PR TITLE
Blood Cell Detection

### PR DESCRIPTION
**Pull request by khushaallll**
# Pull Request for DL-Simplified 💡

## Issue Title : Blood Cell Detection

- **Info about the related issue (Aim of the project)** :  Create a model which will detect the blood cells. 
- **Name:** Khushal Koli
- **GitHub ID:** @khushaallll
- **Email ID:** kolikhki19it@student.mes.ac.in
- **Idenitfy yourself:** -

Closes: #36 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

My project is able to detect different types of cell in blood.
- [ ] The project features data preprocessing and augmentation
- [ ] The project has 3 different CNN models - Custom CNN, VGGNet-16, Xception
- [ ] Pretrained Networks dataset is ImageNet
- [ ] Custom CNN and Xception trained for 100 and 50 epochs respectively with early stop mechanism
- [ ] VGGNet-16 trained for 100 epochs.
- [ ] Graphs have been plotted for visualizations.

## How Has This Been Tested? ⚙️

The training has been validated with loss and accuracy metrics
Results are evident in the graphs and classification

## Checklist: ☑️

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.

### CODE REVIEW

**Comment by abhisheks008** [comment]
@khushaallll 
1. Put the URL of the dataset inside a README.md file in your Dataset folder. No need to put all those datapoints. As it's going to be huge junk.
2. In the Images folder, put the visualization/EDA images only.
3. Add a `requirements.txt` file which contains all the libraries/packages.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Please do the changes .

**Comment by khushaallll** [comment]
Expected Changes Updated

**Comment by abhisheks008** [comment]
No need to put all the `test` and `train` images inside the `Dataset` folder. You have put the URL for the dataset that's enough for others to fetch the datapoints. Remove the `test` and `train` images.
@khushaallll

**Comment by abhisheks008** [comment]
@khushaallll please do the required changes otherwise this will not be counted towards the contribution to this project repository.
